From 540d8f50658d995e01f1fb00f12a534bb7e50272 Mon Sep 17 00:00:00 2001
From: Felipe Contreras <felipe.contreras@gmail.com>
Date: Tue, 23 Mar 2010 21:25:33 +0000
Subject: [PATCH 024/135] dsp-bridge: deh: remove obvious comments

They are weirdly formatted and don't provide any useful information.

Signed-off-by: Felipe Contreras <felipe.contreras@gmail.com>
---
 drivers/dsp/bridge/wmd/ue_deh.c |   25 +------------------------
 1 files changed, 1 insertions(+), 24 deletions(-)

diff --git a/drivers/dsp/bridge/wmd/ue_deh.c b/drivers/dsp/bridge/wmd/ue_deh.c
index 14dd8ae..7a210cd 100644
--- a/drivers/dsp/bridge/wmd/ue_deh.c
+++ b/drivers/dsp/bridge/wmd/ue_deh.c
@@ -60,10 +60,7 @@ static struct hw_mmu_map_attrs_t map_attrs = { HW_LITTLE_ENDIAN,
 #define VIRT_TO_PHYS(x)       ((x) - PAGE_OFFSET + PHYS_OFFSET)
 
 static u32 dummy_va_addr;
-/*
- *  ======== bridge_deh_create ========
- *      Creates DEH manager object.
- */
+
 dsp_status bridge_deh_create(OUT struct deh_mgr **phDehMgr,
 			     struct dev_object *hdev_obj)
 {
@@ -131,10 +128,6 @@ dsp_status bridge_deh_create(OUT struct deh_mgr **phDehMgr,
 	return status;
 }
 
-/*
- *  ======== bridge_deh_destroy ========
- *      Destroys DEH manager object.
- */
 dsp_status bridge_deh_destroy(struct deh_mgr *hdeh_mgr)
 {
 	dsp_status status = DSP_SOK;
@@ -161,10 +154,6 @@ dsp_status bridge_deh_destroy(struct deh_mgr *hdeh_mgr)
 	return status;
 }
 
-/*
- *  ======== bridge_deh_register_notify ========
- *      Registers for DEH notifications.
- */
 dsp_status bridge_deh_register_notify(struct deh_mgr *hdeh_mgr, u32 event_mask,
 				   u32 notify_type,
 				   struct dsp_notification *hnotification)
@@ -184,10 +173,6 @@ dsp_status bridge_deh_register_notify(struct deh_mgr *hdeh_mgr, u32 event_mask,
 	return status;
 }
 
-/*
- *  ======== bridge_deh_notify ========
- *      DEH error notification function. Informs user about the error.
- */
 void bridge_deh_notify(struct deh_mgr *hdeh_mgr, u32 ulEventMask, u32 dwErrInfo)
 {
 	struct deh_mgr *deh_mgr_obj = (struct deh_mgr *)hdeh_mgr;
@@ -305,10 +290,6 @@ void bridge_deh_notify(struct deh_mgr *hdeh_mgr, u32 ulEventMask, u32 dwErrInfo)
 	}
 }
 
-/*
- *  ======== bridge_deh_get_info ========
- *      Retrieves error information.
- */
 dsp_status bridge_deh_get_info(struct deh_mgr *hdeh_mgr,
 			    struct dsp_errorinfo *pErrInfo)
 {
@@ -332,10 +313,6 @@ dsp_status bridge_deh_get_info(struct deh_mgr *hdeh_mgr,
 	return status;
 }
 
-/*
- *  ======== bridge_deh_release_dummy_mem ========
- *      Releases memory allocated for dummy page
- */
 void bridge_deh_release_dummy_mem(void)
 {
 	kfree((void *)dummy_va_addr);
-- 
1.7.0.4

